Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1076,12 +1076,16 @@
   loadouts:
   - HeadofSecurityJumpsuit
   - HeadofSecurityJumpskirt
-  - HeadofSecurityTurtleneck
-  - HeadofSecurityTurtleneckSkirt
+  - HeadofSecurityJumpsuitGrey # DeltaV
+  - HeadofSecurityJumpskirtGrey # DeltaV
+  - HeadofSecurityJumpsuitBlue # DeltaV
+  - HeadofSecurityJumpskirtBlue # DeltaV
+#  - HeadofSecurityTurtleneck
+#  - HeadofSecurityTurtleneckSkirt
```

Comment:
I dont see why these need to be removed from the loadout, perhaps a resprite instead of removal to clean it up?